File: 백준/Bronze/2587. 대표값2/대표값2.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();
        
        int[] arr = new int[5];
        int sum = 0;
        for(int i=0;i<5;i++) {
            int num = Integer.parseInt(br.readLine());
            arr[i] = num;
            sum += num;
        }
        Arrays.sort(arr);
        sb.append(String.valueOf(sum/5) + "\n");
        sb.append(arr[2]);
        System.out.print(sb);
        
        br.close();
    }
}

File: 백준/Bronze/2587. 대표값2/대표값2.py
import sys
fi = sys.stdin.readline
                
def solution():
    num = []
    for i in range(5):
        num.append(int(fi().rstrip()))
    num.sort()
    
    avg = sum(num) // len(num)
    miv = num[2]
    
    print(avg)
    print(miv)
    
if __name__ == '__main__':
    solution()